### PR TITLE
Emj add label filter

### PR DESCRIPTION
**Pull request by emmerentia**
Added the ability to filter on the images that are displayed for a specific task.

### CODE REVIEW

File: frontend/spa/src/components/ImageLabeling.vue [hunk 1]
```diff
@@ -2,134 +2,186 @@
...
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col">
+                                <select width="100" style="width: 100px" id="tools_form" v-on:change="tool_selected" v-model="active_tool">
+                                    <option class="radio-button" name="tool" value="freehand"> Freehand </option>
+                                    <option class="radio-button" name="tool" value="polygon"> Polygon </option>
+                                    <option class="radio-button" name="tool" value="rectangle"> Rectangle </option>
+                                    <option class="radio-button" name="tool" value="point"> Point </option>
+                                    <option class="radio-button" name="tool" value="circle" Circle </option>
```
Comment (simonechan) [inline frontend/spa/src/components/ImageLabeling.vue]: option tag is missing a closing angle bracket - circle is currently not showing on drop down list

File: frontend/spa/src/components/ImageLabeling.vue [hunk 2]
```diff
@@ -263,93 +308,6 @@ export default {
...
-                case 'select':
-                    this.stateNew = true;
-                    this.stateAppend = true;
-                    this.stateErase = true;
-                    this.stateOverlap = true;
-                    this.stateNoOverlap = true;
-                    break;
-            }
-        },
-        disableDrawTools: function() {
```
Comment (simonechan) [inline frontend/spa/src/components/ImageLabeling.vue]: By removing the disableDrawTools function, the functionality of selecting allowed tools for label tasks is also removed. Can we add this functionality back with the drop down list?

Comment (emmerentia) [reply frontend/spa/src/components/ImageLabeling.vue]: Aah, so certain labeling tasks will only allow you to certain drawing tools?

Comment (simonechan) [reply frontend/spa/src/components/ImageLabeling.vue]: Yes

**Comment by simonechan** [comment]
Closes #44

File: frontend/spa/src/components/ImageLabeling.vue [hunk 3]
```diff
@@ -660,25 +632,73 @@ export default {
...
+        defaultDrawTools: function() {
+            this.tools_list = {};
+            this.tools_list["freehand"] = "Freehand";
+            this.tools_list["polygon"] = "Polygon";
+            this.tools_list["rectangle"] = "Rectangle";
+            this.tools_list["point"] = "Point";
+            this.tools_list["circle"] = "Circle";
+        },
+        
+        disableDrawTools: function() {
```
Comment (simonechan) [inline frontend/spa/src/components/ImageLabeling.vue]: The defaultDrawTools and disableDrawTools function is missing the select option. Can we add this back as well?

**Pull request merged**